src/main/pages/che-7/hands-on-guidance/proc_adding-support-for-a-new-language.adoc

```diff
@@ -1,27 +1,39 @@
 [id="adding-support-for-a-new-language_{context}"]
 = Adding support for a new language
 
-This paragraph is the procedure module introduction: a short description of the procedure.
+This section describes how to add support for a new language into Eclipse Che. 
+
+While you can directly hook up a Theia plugin or VS Code extension to the plugin API, the usual way is to write a VS Code extension that uses the languageserver-node library to connect a language server to the VS Code extension API.
 
 [discrete]
 == Prerequisites
 
-* A bulleted list of conditions that must be satisfied before the user starts following this assembly.
-* You can also link to other modules or assemblies the user must follow before starting this assembly.
-* Delete the section title and bullets if the assembly has no prerequisites.
+* Install the latest Che-Theia plug-in Yeoman generator by running:
```

Comment:
Should it also explain from where to start? Using with stack for example?

Reply:
Maybe pointing to: https://github.com/eclipse/che-theia/blob/master/CONTRIBUTING.md